solving-coding-challenges/solving-coding-challenges.md

```diff
@@ -0,0 +1,190 @@
+# Solving Coding Challenges
+
+### Projected Time
+
+65-75 minutes
+- Lesson: 20 min
+- Guided Practice: 15 min
+- Independent Practice: 20 min
+- Check for Understanding: 10 min
+
+### Prerequisites
+
+None
+
+### Motivation
+
+Solving coding challenges is an effective way to practice the concepts you learn.  
+
+While learning a new coding language, solving challenges in that language is helpful for learning syntax, how to represent data structures, and how to debug. 
+
+When learning a new algorithm, a new coding concept, or data structure, applying it to a coding problem is a great way to apply what you've learned and check your understanding. 
+
+Solving coding challenges is also an important skill for job hunting because it shows companies your technical and non-technical skills. They can see how familiar you are with coding, algorithms, etc and they can see how you think, debug, and communicate.
+
+
+### Objectives
+
+**Participants will be able to:**
+- Solve an online coding challenge
+
+### Specific Things To Learn
+At the end of the lesson you will be able to:
+
+- Understand how to solve problems: 
+	- Understanding the prompt
+	- Think of  a high level approach
+	- Code your solution
+	- Test your solution
+
+### Materials
+- [How to think like a programmer](https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/) - High level overview on how to approach coding problems
```

Comment:
For both of these, could you also add the amount of time they should spend on the resource and what part to look at?